### PR TITLE
Fixed enums for payments, invoices and on-chain transactions tables.

### PR DESCRIPTION
**Pull request by ohenrik**
Fixes https://github.com/lncapital/torq/issues/469

NB! We still need to fix the filtering of Arrays and Enums in the processFilter.go file used in the automations!

### CODE REVIEW

File: internal/workflows/processFilter.go [hunk 1]
```diff
@@ -377,6 +377,7 @@ func GetFilterFunctions() map[FilterCategoryType]map[string]FilterFunc {
 			},
 		},
 		FilterCategoryTypeEnum: {
+			// TODO: This needs to be fixed. Enums are using any and notAny instead of like and notLike
```
Comment (ohenrik) [inline internal/workflows/processFilter.go:380]: This needs to accept an array of values and check if the enum value is equal to any of those chosen in the values.

File: internal/workflows/processFilter.go [hunk 2]
```diff
@@ -691,6 +692,7 @@ func GetFilterFunctions() map[FilterCategoryType]map[string]FilterFunc {
 			},
 		},
 		FilterCategoryTypeArray: {
+			// TODO: This needs to be fixed. Arrays are using any and notAny instead of like and notLike
 			"eq": func(dataMap map[string]interface{}, dataKey string, filterValue FilterParameterType) bool {
 				if isNil(dataMap[dataKey]) != (filterValue == nil) {
 					return false
```
Comment (ohenrik) [inline internal/workflows/processFilter.go:698]: This needs to accept an array of values and check if the enum value is equal to any of those chosen in the values.

**Review by ohenrik — COMMENTED** [review]
Enum and Array filtering in the processFilter.go file needs to be fixed

File: web/src/features/sidebar/sections/filter/FilterRow.tsx [hunk 3]
```diff
@@ -182,8 +182,10 @@ function FilterRow({
   const label = filterOptions.find((item) => item.value === rowValues.key)?.label;
   const options = filterOptions.find((item) => item.value === rowValues.key)?.selectOptions || [];
 
-  const getParameter = () => {
+  function getParameter() {
```
Comment (maxwedwards) [inline web/src/features/sidebar/sections/filter/FilterRow.tsx:185]: Why the change in style? I thought all the cool kids liked const funcs and they had better `this` behaviour.

**Comment by maxwedwards** [comment]
Happy for this to be merged when ready

**Comment by ohenrik** [comment]
I'm creating a separate issue for the processFilter.go change needed.
